proto/expr.proto

```diff
@@ -192,10 +198,8 @@ message ExprNode {
     VNODE = 1101;
     // Non-deterministic functions
     PROCTIME = 2023;
-    // User defined functions
-    UDF = 3000;
```

Comment:
Should we keep the `UDF` type and further move `UserDefinedFunction` into the `FunctionCall`? 🤔

Reply:
`FunctionCall` only consists of children and the type, but `UserDefinedFunction` requires more information. 😕

Reply:
But `UserDefinedTableFunction` is embedded in `TableFunction` 🥶

Reply:
Right, they are inconsistent. But it may be unable to change for compatibility. 🥵